### PR TITLE
NO-ISSUE: Change the memory unit from GB to GiB for OCS

### PR DESCRIPTION
**Pull request by rewantsoni**
Signed-off-by: Rewant Soni <resoni@redhat.com>

### CODE REVIEW

File: internal/operators/ocs/validations.go [hunk 1]
```diff
@@ -12,13 +12,13 @@ import (
 
 type Config struct {
 	OCSMinimumCPUCount                int64             `envconfig:"OCS_MINIMUM_CPU_COUNT" default:"18"`
-	OCSMinimumRAMGB                   int64             `envconfig:"OCS_MINIMUM_RAM_COUNT" default:"57"`
+	OCSMinimumRAMGiB                  int64             `envconfig:"OCS_MINIMUM_RAM_COUNT" default:"57"`
```
Comment (priyanka19-98) [inline internal/operators/ocs/validations.go:15]: Please update `OCS_MINIMUM_RAM_COUNT` to `OCS_MINIMUM_RAM_GiB`

Comment (rewantsoni) [reply internal/operators/ocs/validations.go:15]: Done

File: internal/operators/ocs/ocs_operator_test.go [hunk 2]
```diff
@@ -17,31 +17,31 @@ var _ = Describe("Ocs Operator Test GetHostRequirements", func() {
 		ctx      = context.TODO()
 		operator = NewOcsOperator(common.GetTestLog())
 		master1  = &models.Host{Role: models.HostRoleMaster,
-			Inventory: Inventory(&InventoryResources{Cpus: 12, Ram: 32 * conversions.GB,
+			Inventory: Inventory(&InventoryResources{Cpus: 12, Ram: 32 * conversions.GiB,
 				Disks: []*models.Disk{
 					{SizeBytes: 20 * conversions.GB, DriveType: "HDD"},
 					{SizeBytes: 40 * conversions.GB, DriveType: "SSD"},
 				}})}
 		master2 = &models.Host{Role: models.HostRoleMaster, Inventory: Inventory(&InventoryResources{Cpus: 12, Ram: 32})}
```
Comment (nbalacha) [inline internal/operators/ocs/ocs_operator_test.go:25]: Does this need the conversions.GiB factor as well?

File: internal/operators/ocs/validations.go [hunk 3]
```diff
@@ -232,7 +232,7 @@ func getValidDiskCount(disks []*models.Disk) int {
 	var countDisks int
 
 	for _, disk := range disks {
-		if disk.SizeBytes >= conversions.GbToBytes(MinDiskSize) && (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) {
+		if disk.SizeBytes >= conversions.GibToBytes(MinDiskSize) && (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) {
```
Comment (nbalacha) [inline internal/operators/ocs/validations.go:235]: The MinDiskSize has not been changed to GiB in the constant definition. We are still using GB. (See line 220 as well)

File: internal/operators/ocs/validations.go [hunk 4]
```diff
@@ -12,13 +12,13 @@ import (
 
 type Config struct {
 	OCSMinimumCPUCount                int64             `envconfig:"OCS_MINIMUM_CPU_COUNT" default:"18"`
-	OCSMinimumRAMGB                   int64             `envconfig:"OCS_MINIMUM_RAM_COUNT" default:"57"`
+	OCSMinimumRAMGiB                  int64             `envconfig:"OCS_MINIMUM_RAM_GiB" default:"57"`
```
Comment (nbalacha) [inline internal/operators/ocs/validations.go:15]: Will changing the names in the envconfig affect ant tests, for eg, ci?

File: internal/operators/ocs/validations.go (hunk 3 above)
Comment (priyanka19-98) [reply internal/operators/ocs/validations.go:235]: +1

File: internal/operators/ocs/ocs_operator_test.go (hunk 2 above)
Comment (rewantsoni) [reply internal/operators/ocs/ocs_operator_test.go:25]: Yes, I will do that

File: internal/operators/ocs/validations.go (hunk 3 above)
Comment (rewantsoni) [reply internal/operators/ocs/validations.go:235]: Done

File: internal/operators/ocs/validations.go (hunk 4 above)
Comment (rewantsoni) [reply internal/operators/ocs/validations.go:15]: Yes, but they aren't used in CI test

File: internal/operators/ocs/ocs_operator.go [hunk 5]
```diff
@@ -106,7 +106,7 @@ func (o *operator) ValidateHost(ctx context.Context, cluster *common.Cluster, ho
 }
 
 // GenerateManifests generates manifests for the operator
-func (o *operator) GenerateManifests(cluster *common.Cluster) (map[string][]byte, error) {
+func (o *operator) GenerateManifests(_ *common.Cluster) (map[string][]byte, error) {
```
Comment (nbalacha) [inline internal/operators/ocs/ocs_operator.go:109]: Does this need to be changed in this PR?

Comment (rewantsoni) [reply internal/operators/ocs/ocs_operator.go:109]: we aren't using this variable anywhere so I changed it, should I open a different PR for this?

Comment (nbalacha) [reply internal/operators/ocs/ocs_operator.go:109]: yes, it is best to only have the changes relevant to the PR.

Comment (rewantsoni) [reply internal/operators/ocs/ocs_operator.go:109]: Okay, I will update it.

**Review by nbalacha — APPROVED** [review]

**Comment by nbalacha** [comment]
@gamli75 , please review this PR.

File: internal/operators/ocs/validations.go (hunk 4 above)
Comment (gamli75) [reply internal/operators/ocs/validations.go:15]: @rewantsoni Do we need to update QE about it? 
@lalon4 FYI

Comment (rewantsoni) [reply internal/operators/ocs/validations.go:15]: Yes @gamli75, QE can use these envconfig

Comment (gamli75) [reply internal/operators/ocs/validations.go:15]: it means that QE will need to update their tests after this change will be merged to master. Please **don't** push it before 05/05 - the next deployment to production.
@lalon4 are you using that envconfig in your testing?

Comment (lalon4) [reply internal/operators/ocs/validations.go:15]: we use default envconfig to deploy PSI / minikube - so we are good

**Review by gamli75 — APPROVED** [review]

**Pull request merged**